js/video/shared/VideoPreview.svelte

```diff
@@ -65,9 +65,9 @@
 	{/key}
 	<div class="icon-buttons" data-testid="download-div">
 		<a
-			href={value.path}
+			href={value.url}
 			target={window.__is_colab__ ? "_blank" : null}
-			download={value.orig_name || value.path}
+			download={value.orig_name || value.url}
```

Comment:
I think this should just be value.url like it is for `StaticAudio`?

Reply:
Looked this up because I wasn't sure. From https://www.w3schools.com/tags/att_a_download.asp:

> value of the download attribute will be the new name of the file after it is downloaded

So I actually think `value.orig` is the appropriate one. I'm not sure if the latter parameter is ever used, but if anything, it should probably stay `value.path`. I'll let @hannahblair change this if needed.

Reply:
Yea read into it and it's the name the file will be given when downloaded. So `orig_name` or `path` is the way to go.